blsqpy/extract.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+from blsqpy.descriptor import Descriptor
+
+
+def to_data_elements(activity):
+    data_elements = []
+    for _state_code, state in Descriptor.as_items(activity.states):
+        for _source_code, source in Descriptor.as_items(state.sources):
+            data_elements.extend(source.uids)
+    return data_elements
+
+
+def rotate_de_coc_as_columns(df):
+    df = df.rename(index=str, columns={
```

Comment:
Let's standardize our naming earlier (on df or config) so we stay consistent in naming accross the pipe